nn_sdwan/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "display_name": "nn-sdwan",
+  "maintainer": "info@netnology.io",
+  "manifest_version": "1.0.0",
+  "name": "nn_sdwan",
+  "metric_prefix": "nn_sdwan.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "SDWAN Controller Metric Exporter by Netnology",
+  "guid": "a7c79ba9-a4b7-4fa4-b4aa-3868458e2101",
+  "support": "extras",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "nn-sdwan",
+  "categories": [
```

Comment:
If you want to add categories to your listing, you can find the list at the top of this page: [https://docs.datadoghq.com/integrations/](https://docs.datadoghq.com/integrations/)